recipe/build.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./build-aux
```

Comment:
Would running `autoreconf` do this for us?

Reply:
yes, but this was the easiest way to do it automatically in the bot as it's not easy easy to figure out where to run it and what dependencies are needed to run autoreconf.

Reply:
Makes sense.